app/assets/javascripts/discourse/app/components/sidebar/sections.js

```diff
@@ -0,0 +1,17 @@
+import GlimmerComponent from "discourse/components/glimmer";
+import { customSections as sidebarCustomSections } from "discourse/lib/sidebar/custom-sections";
+
+export default class SidebarSections extends GlimmerComponent {
+  customSections;
+
+  constructor() {
+    super(...arguments);
+    this.customSections = this._customSections;
```

Comment:
Why is this needed? Can't you just do `get customSections() {}` below and use that directly in the component?

Reply:
If I use a getter, we have to compute the getter each time the getter is called. I'm using a public field here as a form of memoization, since the public field is only assigned once.

Reply:
I thought you can just use ` @computed` for that with no arguments, no?

Reply:
`@computed` is no longer valid in Glimmer components. Technically, `@computed` with no arguments is also not really valid because it is not considered a computed property when it does not depend on other properties.